src/plugins/intel_gpu/src/runtime/ocl/ocl_common.hpp

```diff
@@ -64,5 +64,30 @@ inline void rethrow_or_exit(const cl::Error& error, const device_info& info) {
     rethrow_or_exit(OCL_ERR_MSG_FMT(error), error.err(), info);
```

Comment:
rethrow_or_exit is not used anymore thus should be removed

Reply:
Removed